analyzer/evmtokens/evm_tokens.go

```diff
@@ -191,8 +191,7 @@ func (m Main) Start() {
 	backoff, err := util.NewBackoff(
 		100*time.Millisecond,
 		6*time.Second,
-		// ^cap the timeout at the expected
-		// emerald round time
+		// ^cap the timeout at the expected emerald round time
```

Comment:
`emerald` specifically? Will we use this for sapphire?

Reply:
Not emerald specific, thanks. Lemme fix before merging the PR.